src/Utility/Text.php

```diff
@@ -195,8 +195,6 @@ public static function insert($str, $data, array $options = [])
             return ($options['clean']) ? static::cleanInsert($str, $options) : $str;
         }
 
-        asort($data);
```

Comment:
This was added in bf2829a8 which predates us having any test coverage. I suspect it is safe to remove if the existing tests pass.